jobs/options.go

```diff
@@ -0,0 +1,24 @@
+package jobs
+
+import "net/url"
+
+type WorkerPoolOption func(*WorkerPool)
+
+func WithJobStatusWebhook(u string) WorkerPoolOption {
+	return func(wp *WorkerPool) {
+		if u == "" {
+			return
+		}
+
+		valid, err := url.ParseRequestURI(u)
+		if err != nil {
+			panic(err)
```

Comment:
This could have some more info (e.g. "invalid webhook URL"), as this currently panics w/ the following error message for example;

```
panic: parse "invalid-url-example": invalid URI for request
```